### ISSUE TITLE
Add option to avoid direct .value changes from triggering event handlers

### ISSUE THREAD

**Issue by DRocksCoding** [alphagov/accessible-autocomplete#421]
Hello,
I've been trying to implement the accessible-autocomplete inside a html table which has 3 instances of accessible-autocomplete per row (one instance for each column).

What I'm trying to do is to set the two other column's instances of autocomplete input.value with javascript with a custom **onConfirm:function(){... input.value = "newValue"}**

The problem arise when the value gets injected because all events trigger at the same time when autoselecting with the "Tab" control. 
For reference here you can see the result of trying to change CATEGORIE + TAXE from the onConfirm of NOM:
[https://i.imgur.com/1ny6TJJ.png](url)
![chrome_PO7UUtjR7M](https://user-images.githubusercontent.com/817459/75211209-492ee900-5751-11ea-9552-36a47c708487.png)
The CATEGORIE column should have been "Honoraires professionnels" but it was skipped probably because a focusin event happenned so fast on it just before setting TAXE to "QC".
The other problem is that the menu--overlay is showing up too.

The desired result would be something like this:
[https://i.imgur.com/78XAKDE.png](url)
![chrome_hZu18Rn8Y5](https://user-images.githubusercontent.com/817459/75211416-d114f300-5751-11ea-9b63-67db65c067b0.png)

I tried disabling the other input before changing the values. I also tried to make _listeners = "" while changing values too.
In any case it doesn't handle it well because there seems to be too many thing happening at once.

Bottom line, would it be possible to have an option allows direct value changes without triggering all the bound event handlers please ?

PS: I cannot edit the source code, I'm just starting out in Javascript

Thanks in advance for any hints

EDIT: I managed to be able to get a bit of success by creating empty input fields before instantiating autocompletes in my <td> cells to be able to create the instance with a default value, but its very messy and hard to manage.

**Comment by ediblecode** [comment]
+1 for this, we've encountered a similar problem.

We're using this component for a search input box, and we're using the history API to change the URL of the page dynamically, for example to _/search?q=text._. When a user navigates using the browser back/forward buttons we're grabbing the q parameter value from the querystring and updating `.value` on the input, which triggers the autocomplete suggestions to load.

**Comment by ediblecode** [comment]
@DRocksCoding maybe this is a duplicate of #250?

**Comment by DRocksCoding** [comment]
> @DRocksCoding maybe this is a duplicate of #250?

I had not seen that one but you are right, I think its a similar request. Both can complement well in my opinion.

**Comment by vanitabarrett** [comment]
I'm going to close this issue as I believe it's a duplicate of https://github.com/alphagov/accessible-autocomplete/issues/250
